app/src/test/java/org/simple/clinic/registration/register/RegistrationLoadingScreenControllerTest.kt

```diff
@@ -36,34 +55,39 @@ class RegistrationLoadingScreenControllerTest {
 
   @Test
   fun `when retry button is clicked, then loader should show and register api should be called`() {
-    whenever(userSession.register()).doReturn(Single.just<RegistrationResult>(Success))
+    whenever(userSession.loggedInUser()) doReturn Observable.just<Optional<User>>(Just(user))
+    whenever(facilityRepository.currentFacility(user)) doReturn Observable.just(facility)
+    whenever(registerUser.registerUserAtFacility(user, facility)).doReturn(Single.just<RegistrationResult>(Success))
```

Comment:
`doReturn` instead of `.doReturn`? Mismatch in convention?

Reply:
`doReturn` is an infix function. I've generally been using `doReturn` in the infix style when writing new stubs but using the normal method style when converting older `thenReturn` statements. I can change this to match the convention.

I could change all of them to use the infix style, or change the newer infix calls to the normal method styles. Do you have any particular preference?

Reply:
I don't really have a preference. Just the first two calls have an `infix` syntax and the third one has the `.` syntax. We could just make them consistent.